core/src/angle.rs

```diff
@@ -237,3 +238,9 @@ impl PartialOrd<f32> for Radians {
         self.0.partial_cmp(other)
     }
 }
+
+impl Display for Radians {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{}", self.0)
```

Comment:
We should probably the unit of measurement here:

```suggestion
        write!(f, "{} rad", self.0)
```

Reply:
The issue I see here is whenever a user does not wish to include the unit of measurement as removing it is more complex than having it gone by default and adding it whenever it's explicitly used/needed.